### PR TITLE
Update alert history chart screenshots

### LINKED ISSUE #4129: [Request]: Update alert history chart screenshots
### Description

In this [ticket](https://github.com/elastic/kibana/issues/181475), we changed the alert history chart and now it looks like this: ([example](https://edge-oblt.kb.us-west2.gcp.elastic-cloud.com/app/observability/alerts/ede0f747-e628-4426-af77-08091ec545eb))

image

We need to update pages that have screenshots from the old view, here is the list of pages that I could find:
- [Triage threshold breaches](https://www.elastic.co/guide/en/observability/current/triage-threshold-breaches.html)
- [Triage SLO burn rate breaches](https://www.elastic.co/guide/en/observability/current/triage-slo-burn-rate-breaches.html)

### Resources

- Ticket: https://github.com/elastic/kibana/issues/181475
- PR: https://github.com/elastic/kibana/pull/181824

### Which documentation set does this change impact?

Stateful and Serverless

### Feature differences

No difference

### What release is this request related to?

8.15

### Collaboration model

The documentation team

### Point of contact.

**Main contact:** @maryam-saeidi 

**Stakeholders:** Obs-ux-management team

### PR DESCRIPTION
**Pull request by colleenmcginnis**
## Description

Update alert history chart screenshots in the _Triage SLO burn rate breaches_ ([preview](https://observability-docs_bk_4301.docs-preview.app.elstc.co/guide/en/observability/master/triage-slo-burn-rate-breaches.html)) and _Triage threshold breaches_ ([preview](https://observability-docs_bk_4301.docs-preview.app.elstc.co/guide/en/observability/master/triage-threshold-breaches.html)) pages.

This PR also makes a few other changes:

* Splits screenshots into individual images for each chart to make it easier to maintain.
* Attempts to make the list describing charts more browsable with bold text at the beginning.
* Moves the last bullet point about timeline annotations to the first bullet point on both pages because (I think) with the change to the alert history chart, there are no longer annotations in alerts history.

### Documentation sets edited in this PR

_Check all that apply._

- [x] Stateful (`docs/en/observability/*`)
- [ ] Serverless (`docs/en/serverless/*`)
- [ ] Integrations Developer Guide (`docs/en/integrations/*`)
- [ ] None of the above

### Related issue

Closes #4129

## Checklist

- [x] Product/Engineering Review
- [x] Writer Review
    (@dedemorton I'd be particularly interested in your opinion of the changes since I _think_ you wrote the initial content)

### Follow-up tasks

_Select one._

* This PR does _not_ need to be ported to another doc set because:
  - [ ] The concepts in this PR only apply to one doc set (serverless _or_ stateful)
  - [ ] The PR contains edits to both doc sets (serverless _and_ stateful)
* This PR needs to be ported to another doc set:
  - [ ] Port to stateful docs: \<link to PR or tracking issue>
  - [x] Port to serverless docs: https://github.com/elastic/observability-docs/pull/4320

### CODE REVIEW

File: docs/en/observability/triage-threshold-breaches.asciidoc [hunk 1]
```diff
@@ -19,22 +19,35 @@ You can follow the links to navigate to the rule definition.
...
 For more information about using the log rate analysis feature,
 refer to the {kibana-ref}/xpack-ml-aiops.html#log-rate-analysis[AIOps Labs] documentation.
-* The page may also include an alerts history chart that shows the number of triggered alerts per day for the last 30 days.
++
+[role="screenshot"]
+image::images/log-threshold-breach-log-rate-analysis.png[Log rate analysis chart in alert details for log threshold breach]
+
+* *Alerts history chart*. The page may also include an alerts history chart that shows the number of
+triggered alerts per day for the last 30 days.
 This chart is currently only available for rules that specify a single condition.
```
Comment (maryam-saeidi) [inline docs/en/observability/triage-threshold-breaches.asciidoc:47]: This part can be removed, as mentioned above.

File: docs/en/observability/triage-slo-burn-rate-breaches.asciidoc [hunk 2]
```diff
@@ -18,15 +18,23 @@ You can follow the links to navigate to the source SLO or rule definition.
...
+[TIP]
+====
+The timeline is annotated to show when the threshold was breached.
+You can hover over an alert icon to see the timestamp of the alert.
+====
 
-* The first chart shows the burn rate during the time range when the alert was active.
-The line indicates how close the SLO came to breaching the threshold.
-* The next chart shows the alerts history over the last 30 days.
+* *Alerts history chart*. The next chart shows the alerts history over the last 30 days.
```
[the remark this answers is not in the document]
Comment (colleenmcginnis) [reply docs/en/observability/triage-slo-burn-rate-breaches.asciidoc]: Thanks @maryam-saeidi! I pushed some edits in https://github.com/elastic/observability-docs/pull/4301/commits/584670d6895ae15e877e1021d46e19dc2523577a. Can you take another look when you get a chance?

[the remark this answers is not in the document]
Comment (jasonrhodes) [reply docs/en/observability/triage-slo-burn-rate-breaches.asciidoc]: Nothing else specific from me, thanks.

[the remark this answers is not in the document]
Comment (maryam-saeidi) [reply docs/en/observability/triage-slo-burn-rate-breaches.asciidoc]: Thanks, @colleenmcginnis, for the update!

**Review by maryam-saeidi — APPROVED** [review]
LGTM!

**Review by bmorelli25 — APPROVED** [review]
looks good from my perspective too

**Pull request merged**